website/docs/reference/dbt-jinja-functions/builtins.md

```diff
@@ -15,9 +15,43 @@ The `builtins` variable is a dictionary containing the following keys:
 
 ## Usage
 
-The following macro overrides the `ref` method available in the model compilation context to return a [Relation](/reference/dbt-classes#relation) with the database name overriden to `dev`.
+<VersionBlock firstVersion="1.6">
+  
+From dbt v1.6 and higher, use the following macro to extract user-provided arguments, including <code>version</code>, and call the <code>builtins.ref()</code> function with either a single <code>modelname</code> argument or both <code>packagename</code> and <code>modelname</code> arguments, based on the number of positional arguments in <code>varargs</code>:
 
+<br /><br />
+
+  
+```jinja2
+jinja2.ext.do
```

Comment:
I don't think we should include this `jinja2.ext.do`, because a dbt user wouldn't need/want to include it when copy-pasting this code. Did you find that it was necessary to support the syntax highlighting? If so, I would just remove `jinja2` above as well.

Reply:
ok gotcha, thanks for clarifying that @jtcohen6. i was getting an error when  it managed to help me pass the checks. now that I've removed it, it's passing 🤦‍♀️ so idk what happened on fri